### DISCUSSION TITLE
My Cameras

### DISCUSSION THREAD

**Discussion by easytarget** [easytarget/esp32-cam-webserver#164] [Show and tell]
Thought I'd post my own setup here; currently I have three modules (the 4th failed to survive my putting 5V on the 3V3 pin)

### One is mounted as a monitor webcam in my LaserCutter:
![IMG_20210923_215320](https://user-images.githubusercontent.com/4005142/134591744-33855e0b-701f-457a-bef4-28098b7d6fd6.jpg)
- This is a wide-angle cam

### Another is in my Printroom:
 ![IMG_20210923_220943](https://user-images.githubusercontent.com/4005142/134591922-d585250e-cd33-4f67-be84-733b984581a6.jpg)
- Also a wide-angle cam

### The last one is permanently attached to my test-rig and is used for development:
![IMG_20210924_003025](https://user-images.githubusercontent.com/4005142/134592812-a192a317-6b09-4bac-882c-b9ec2c572ac0.jpg)

**Comment by easytarget** [comment]
Here is the view you get from the printroom camera:
![printroom-cam](https://user-images.githubusercontent.com/4005142/134664500-7296abb6-594e-42fd-8bee-f4a468d81d27.jpg)
And the view inside the lasercutter
![hyacinth-cam](https://user-images.githubusercontent.com/4005142/134664584-d6b388af-c05e-4916-b58d-76bf174e8ffa.jpg)
Finally, I made an elephant to hold my development board (I've got a lasercutter..)
![IMG_20210924_131256](https://user-images.githubusercontent.com/4005142/134665830-ff298606-b623-4f98-ba46-de248666ca0b.jpg)

**Reply by Tecnhotron** [reply]
Nice cameras sir

**Comment by moelect** [comment]
I am envious of your great collection of equipment!

**Comment by RC-Aiden** [comment]
Out of curiosity... considering your capturing in UXGA ( 1600x1200 )... what fps do you get when you stream the web cam?
And what have you found to be the best performing but the most stable settings for your setup?

**Comment by easytarget** [comment]
Settings are all default, except resolution of 1280x764 and AGC gain raised on both cameras, I hardly ever stream them but when I do the framerate is low, and varies wildly.
I flicked them to max resolution just for taking the still images here.
In normal mode of operation they take an image once per minute, which is saved on my server. I don't run these machines unattended or very frequently, so the cameras only get used occasionally.

**Comment by easytarget** [comment]
Here is an interesting dump, from the camera in my workshop:
![Screenshot from 2022-03-14 14-11-53](https://user-images.githubusercontent.com/4005142/158178774-1e73dad7-001b-4bf8-9714-e8b103c9c6c0.png)

Note the 62 days of uptime! and total images captured. (my server grabs+archives an image once per minute if it can)
This has also survived numerous wifi disconnects in this time. And is currently (successfully) streaming to my server for over a day now as part of a load test.
